WireAuthentication/Sources/WireAuthentication/Components/PersonalAccountCreationComponent.swift

```diff
@@ -25,14 +25,17 @@ internal import WireAuthenticationLogic
 final class PersonalAccountCreationComponent: Component<PersonalAccountCreationComponentDependency> {
 
     private let email: String
+    private let backendURL: URL
     private let teamAccountCreationLink: URL?
 
     init(
         parent: any Scope,
         email: String,
+        backendURL: URL,
```

Comment:
Injecting the backendURL so that we can compare the host values with the hard-coded prod and staging values in order to enable analytics.